### PR TITLE
Update fonts to match website

### PR DESCRIPTION
**Pull request by nathunsmitty**
Taking care of issue #7.

Some screenshots!
screen shot 2017-02-15 at 4 52 54 pm
screen shot 2017-02-15 at 4 52 39 pm
screen shot 2017-02-15 at 4 52 34 pm
screen shot 2017-02-15 at 4 50 05 pm
screen shot 2017-02-15 at 4 50 00 pm
screen shot 2017-02-15 at 4 54 30 pm
screen shot 2017-02-15 at 4 54 26 pm

### CODE REVIEW

File: UCLA Radio/UCLA Radio/Info.plist [hunk 1]
```diff
@@ -15,11 +15,11 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>2.2</string>
+	<string>2.1</string>
```
Comment (ChrisLaganiere) [inline UCLA Radio/UCLA Radio/Info.plist]: Can you remove this change?

File: UCLA Radio/UCLA Radio/Info.plist [hunk 2]
```diff
@@ -15,11 +15,11 @@
...
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>7</string>
+	<string>4</string>
```
Comment (ChrisLaganiere) [inline UCLA Radio/UCLA Radio/Info.plist]: and this one

File: UCLA Radio/UCLA Radio/NowPlayingViewController.swift [hunk 3]
```diff
@@ -100,6 +100,16 @@ class NowPlayingViewController: UIViewController, HistoryFetchDelegate, SlidingV
...
+        
+        for family in UIFont.familyNames
+        {
+            print("\(family)")
+            for names: String in UIFont.fontNames(forFamilyName: family)
+            {
+                print("== \(names)")
+            }
+        }
+
```
Comment (ChrisLaganiere) [inline UCLA Radio/UCLA Radio/NowPlayingViewController.swift]: test block?

File: UCLA Radio/UCLA Radio/menu/MenuViewController.swift [hunk 4]
```diff
@@ -93,6 +93,7 @@ class MenuViewController: UIViewController, UITableViewDataSource, UITableViewDe
         let viewController = UIStoryboard(name: "Main", bundle: nil).instantiateViewController(withIdentifier: storyboardID)
         if let navigationController = navigationController {
             navigationController.pushViewController(viewController, animated: true)
+            print("hey")
```
Comment (ChrisLaganiere) [inline UCLA Radio/UCLA Radio/menu/MenuViewController.swift]: hey

File: UCLA Radio/UCLA Radio/menu/MenuViewController.swift [hunk 5]
```diff
@@ -143,6 +144,7 @@ class MenuViewController: UIViewController, UITableViewDataSource, UITableViewDe
         
         let item = items[(indexPath as NSIndexPath).row]
         if let storyboardID = item.storyboardID {
+            print("hi")
```
Comment (ChrisLaganiere) [inline UCLA Radio/UCLA Radio/menu/MenuViewController.swift]: hi

**Comment by nathunsmitty** [comment]
All changes should be changed! 👍 Good reminder on me to delete those superfluous `print` statements.

**Pull request merged**